lib/pay/merchant.rb

```diff
@@ -2,6 +2,17 @@ module Pay
   module Merchant
     extend ActiveSupport::Concern
 
+    # Keep track of which Merchant models we have
+    class << self
```

Comment:
This was copied from the billiable class. it probably would be good to abstract even though I don't think it will change much

Reply:
Yeah, I just ripped this out of Madmin and I'm wondering if we may want to do the same here.

This is nice, but only is reliable if you've eager loaded your whole app. It may make more sense to have a configuration of `merchant_models = ["User", "Account"]`

We can roll with this for now though.